.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -0,0 +1,45 @@
+name: "\U0001F41B Bug report"
+description: Create a report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue.
+
+        This issue tracker is for bugs and issues found within Node.js core.
+        If you require more general support please file an issue on our help repo. https://github.com/nodejs/help
+
+        Please fill in as much of the form below as you're able.
+  - type: input
+    attributes:
+      label: Version
+      description: Output of `node -v`
+  - type: input
+    attributes:
+      label: Platform
+      description: |
+        UNIX: output of `uname -a`
+        Windows: output of `"$([Environment]::OSVersion | ForEach-Object VersionString) $(if ([Environment]::Is64BitOperatingSystem) { "x64" } else { "x86" })"` in PowerShell console
+  - type: input
+    attributes:
+      label: Subsystem
+      description: If known, please specify affected core module name
```

Comment:
For this field, it'd be nice to have a `<select multiple>`, but it can happen in another PR.

Reply:
I thought about it and was hesitant. Let's keep it out of this PR if everyone agrees.